src/test/java/organice/logic/parser/AddressBookParserTest.java

```diff
@@ -70,10 +70,12 @@ public void parseCommand_exit() throws Exception {
 
     @Test
     public void parseCommand_find() throws Exception {
-        List<String> keywords = Arrays.asList("foo", "bar", "baz");
+        String searchParams = "find n/Alice ic/S1111111A t/doctor";
```

Comment:
Should use constant but its ok for now like `commandText = FindCommand.Command_Word .....`

Reply:
changed